From b81e2238c477a0b0be200eb9a59b767fca0b7757 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:23:22 -0400
Subject: [redhat] kabi: Adding symbol flush_signals (kernel/signal.c)

Message-id: <b556c222b534dd7aed85adf4c03c83f0b1858f63.1410510773.git.poros@redhat.com>
Patchwork-id: 96032
O-Subject: [PATCH RHEL7.1 082/233] kabi: Adding symbol flush_signals (kernel/signal.c)
Bugzilla: 1071098
RH-Acked-by: Larry Woodman <lwoodman@redhat.com>

Bugzilla: 1071098
    https://bugzilla.redhat.com/show_bug.cgi?id=1071098

    Description:
    ============
    Clear the pre-existing signals before waiting on the queue.
    GAB blocks a single thread for FFDC. It needs to ignore.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/flush_signals
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/flush_signals
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/flush_signals

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/flush_signals b/redhat/kabi/kabi-module/kabi_ppc64/flush_signals
new file mode 100644
index 0000000..be964c5
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/flush_signals
@@ -0,0 +1,2 @@
+#1-
+0xfe51a31b flush_signals vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/flush_signals b/redhat/kabi/kabi-module/kabi_s390x/flush_signals
new file mode 100644
index 0000000..088723c
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/flush_signals
@@ -0,0 +1,2 @@
+#1-
+0x001eb4a4 flush_signals vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/flush_signals b/redhat/kabi/kabi-module/kabi_x86_64/flush_signals
new file mode 100644
index 0000000..e695759
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/flush_signals
@@ -0,0 +1,2 @@
+#1-
+0xc5ed6853 flush_signals vmlinux EXPORT_SYMBOL
-- 
1.7.1